app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,36 @@
+/* universal */
+body {
+    padding-top: 60px;
+}
+
+section {
+    overflow: auto;
+}
+
+textarea {
+    resize: vertical;
+}
+
+.center {
+    text-align: center;
+}
+
+.center h1 {
+    margin-bottom: 10px;
+}
+
+/* header */
+#logo {
+    float: left;
+    margin-right: 10px;
+    font-size: 1.7em;
+    color: #fff;
```

Comment:
please create color variable in other file, then call back here

Reply:
may I know which file should I create color variable?